test/run/init_tests.js

```diff
@@ -0,0 +1,22 @@
+const { exec } = require( 'child_process' );
+const { E2E_TESTS } = require( './paths' );
+
+function initTests() {
+    const args = '--timeout 20000';
+    const cmd = `npx mocha "${E2E_TESTS}" ${args}`;
+
+    return new Promise( ( resolve, reject ) => {
+        let tests = exec( cmd, ( error, stdout, stderr ) => {
+            if ( error ) {
+                reject( error );
+                return
+            } else {
+                resolve( stdout? stdout : stderr );
+            }
+        } );
+        tests.stdout.pipe( process.stdout );
+        tests.stderr.pipe( process.stderr );
+    } );
```

Comment:
given my comment about the process runner, this one almost seems like it would be fine inline in the main test runner module. it's ugly, but if we remove some of the variables here it's not so bad, especially since we remove a level of `Promise`

```js
async function run() {
  …

  // run the tests
  const testPath = path.join( __dirname, '../../test/tests/e2e.js' );

  // by default should pipe stdout and stderr to this parent process
  // will throw an error on exception
  execSync( `npx mocha ${ testPath } --timeout 20000` );
}
```

Reply:
Great suggestion. Refactored inline with `execSync` [here](https://github.com/Automattic/wp-desktop/pull/778/files#diff-95f02e568cfd6b3a0b1830f8fd2306a1R84). We have to add the additional `stdio: 'inherit` option so we can leverage streamed (and not buffered) output to the current shell, and it works great. 👍